app/helpers/travel_advice_helper.rb

```diff
@@ -3,9 +3,11 @@
 module TravelAdviceHelper
 
   def group_by_initial_letter(countries)
-    countries.group_by do |country|
+    groups = countries.group_by do |country|
       country.name[0] if country and country.name
     end
+
+    groups.sort_by { |name, _| name }
```

Comment:
No,

`sort_by(&:name)` is equivalent to
`#sort_by { |element| element.name }` which is not the same as
`#sort_by { |name, _| name }`